profiler/build.gradle

```
@@ -0,0 +1,31 @@
+plugins {
+    id("com.github.johnrengelman.shadow") version "6.1.0"
+}
+
+apply from: "$rootDir/gradle/shadow.gradle"
+
+def relocatePackages = ext.relocatePackages
+
+dependencies {
+    compileOnly("io.opentelemetry:opentelemetry-sdk:${versions.opentelemetry}")
+    compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:${versions.opentelemetryAlpha}")
+    compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-spi:${versions.opentelemetryJavaagentAlpha}")
```

`javaagent-spi` does not exist anymore:

```suggestion
    compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-extension-api:${versions.opentelemetryJavaagentAlpha}")
```

---

Yeah, I caught that the hard way.  :). Thanks.  I've updated it.